libs/SalesforceSDKCore/SalesforceSDKCoreTests/CSFSalesforceOAuthRefreshTests.m

```diff
@@ -73,7 +74,8 @@ @interface CSFSalesforceOAuthRefreshTests : XCTestCase
 @implementation CSFSalesforceOAuthRefreshTests
 
 - (void)testRevokedToken {
-    SFUserAccount *user = [SFUserAccount new];
+    SFUserAccount *user = [[SFUserAccountManager sharedInstance]  createUserAccount:[[SFAuthenticationManager sharedManager] createOAuthCredentials]];
```

Comment:
This should be reworked to the spirit of the method.  Create the OAuth credentials as standalone with the logic used to do so below this, then instantiate a user account based on that credentials configuration.